crates/pessimistic-proof/src/imported_bridge_exit.rs

```diff
@@ -15,19 +16,17 @@ pub struct ImportedBridgeExit {
     /// Need to verify that the destination network matches the current network, and that
     /// the bridge exit is included in an imported LER
     pub bridge_exit: BridgeExit,
-
     /// The Network ID for the sending network. May not be strictly
     /// necessary depending on the final structure of the Imported Local Exit Root
     pub sending_network: NetworkId,
-
     /// The Imported Local Exit Root for the Local Exit Tree containing this bridge exit.
     pub imported_local_exit_root: Digest,
-
-    /// The index of the bridge exit in the sending local exit tree. Used to compute the path
-    pub leaf_index: u32,
-
-    /// The inclusion proof of the imported bridge exit in the sending local exit root
+    /// The inclusion proof of the imported bridge exit in the sending local exit root.
     pub inclusion_proof: LETMerkleProof<Keccak256Hasher>,
+    /// The inclusion proof of the LER to the Rollup Exit Root if from rollup.
+    pub inclusion_proof_rer: Option<(LETMerkleProof<Keccak256Hasher>, Digest)>,
```

Comment:
nit: Maybe clarify in the comment that the `Digest` is the rollup exit root

Reply:
Done